css-overrides/Arabic_RTL_Override.css

```
@@ -0,0 +1,90 @@
+body {
+font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+/* Right menu - Hide the scrollbar...*/
```

Stylelint (17.12.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)